iroha_p2p/src/peer.rs

```diff
@@ -160,33 +176,121 @@ where
     // creates shared secret and sends our public key to client
     async fn read_client_hello(&mut self) -> Result<(), Error> {
         debug!("Reading client hello...");
-        #[allow(clippy::unwrap_used)]
-        let read_half = self.read.as_mut().unwrap();
+        #[allow(clippy::expect_used)]
+        let read_half = self
+            .read
+            .as_mut()
+            .expect("Never fails as in this function we already have the stream...");
```

Comment:
I meant `...` as in add any other details that you want, as I don't know the whole context.

Reply:
If there are no other details you can just put `.` :smile: